index.md

```diff
@@ -1,26 +1,15 @@
-Anbox Cloud provides a rich software stack that enables you to run Android applications in the cloud for all kinds of different use cases, including high-performance streaming of graphics to desktop and mobile client devices.
+Anbox Cloud is a cloud platform that containerises mobile workloads using Android as a guest operating system.
 
-Using container technology, Anbox Cloud is scalable from a single-machine installation that can run scores of single Android systems to an entire cluster of machines. It is based on powerful and battle-proven software from Canonical like [LXD](https://linuxcontainers.org/) and [Juju](https://jujucharms.com/).
+It provides a rich software stack that enables you to run Android applications in the cloud. Anbox Cloud manages the container and application life cycle while optimising for high density, performance and fast container boot times.
 
-## Core features
+Using container technology, Anbox Cloud is scalable from a single-machine installation that can run scores of single Android systems to an entire cluster of machines. Deploying the solution is simple and straightforward, and it is based on powerful and battle-proven software from Canonical like [LXD](https://linuxcontainers.org/) and [Juju](https://jujucharms.com/).
```

Comment:
I would like this to address the question of _why it exists_ more pointedly. What problem does Anbox Cloud solve? What was difficult, expensive or impossible until it existed? What pain has it relieved?

(As it stands, this paragraph seems to me to belong with the previous one: it says what Anbox Cloud does, not what need it meets.)

Reply:
How about:
```suggestion
Other solutions usually use virtual machines to emulate Android systems. Anbox Cloud, however, uses system containers, which results in an isolation level similar to virtual machines with an overhead as low as process containers. Therefore, Anbox Cloud can provide at least twice the container density compared to Android emulation in virtual machines, and can serve up to 100 Android instances per server.
```

Reply:
As we plan to add virtual machine support at some point in the future we should reword here to not speak against using VMs (as there is a use case to have VMs). Instead leads emphasize that we concentrate on using system containers with LXD to deliver the best performance/density while still providing security and strict isolation.

Reply:
Tried to make it a bit more general.